src/setup.js

```diff
@@ -196,8 +191,6 @@ if (currentScript) {
         stage:              currentScript.getAttribute('data-stage'),
         apiStage:           currentScript.getAttribute('data-api-stage'),
         stageUrl:           currentScript.getAttribute('data-stage-url'),
-        localhostUrl:       isPayPalDomain() ? currentScript.getAttribute('data-localhost-url') : undefined,
```

Comment:
I think we need to keep support for these `data-localhost-url` and `data-checkout-uri` attributes. They are safe to keep because they have the `isPayPalDomain()` check in there.